app/assets/javascripts/multirange.js

```diff
@@ -0,0 +1,88 @@
+(function() {
+"use strict";
+
+var supportsMultiple = self.HTMLInputElement && "valueLow" in HTMLInputElement.prototype;
+
+var descriptor = Object.getOwnPropertyDescriptor(HTMLInputElement.prototype, "value");
+
+self.multirange = function(input) {
+	if (supportsMultiple || input.classList.contains("multirange")) {
+		return;
+	}
+
+	var value = input.getAttribute("value");
+	var values = value === null ? [] : value.split(",");
+	var min = +(input.min || 0);
+	var max = +(input.max || 100);
+	var ghost = input.cloneNode();
+
+	input.classList.add("multirange", "original");
+	ghost.classList.add("multirange", "ghost");
+
+	input.value = values[0] || min + (max - min) / 2;
+	ghost.value = values[1] || min + (max - min) / 2;
+
+
+	input.parentNode.insertBefore(ghost, input.nextSibling);
+
+	Object.defineProperty(input, "originalValue", descriptor.get ? descriptor : {
+		// Fuck you Safari >:(
+		get: function() { return this.value; },
+		set: function(v) { this.value = v; }
+	});
+
+	Object.defineProperties(input, {
+		valueLow: {
+			get: function() { return Math.min(this.originalValue, ghost.value); },
+			set: function(v) { this.originalValue = v; },
+			enumerable: true
+		},
+		valueHigh: {
+			get: function() { return Math.max(this.originalValue, ghost.value); },
+			set: function(v) { ghost.value = v; },
+			enumerable: true
+		}
+	});
+
+	if (descriptor.get) {
+		// Again, fuck you Safari
```

Comment:
and this

Reply:
Sorry about that. Didn't notice these comments.